style.css

```diff
@@ -446,3 +467,93 @@ hr {
   background-color: #6070ff;
   color: white;
 }
+
+footer {
+  background-color: #6070ff;
+  background-image: url("contact\ form\ shapes-mobile.png");
+  background-size: 375px;
+  top: 5410px;
+  width: 375px;
+  height: 779px;
+  fill: solid;
+}
+
+form {
+  width: 375px;
+  height: 779px;
+  top: 5410px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  justify-content: space-around;
+  align-items: center;
+}
+
+footer legend {
+  color: #fff;
+  font-weight: 700;
+  font-size: 40px;
+  text-align: center;
+  font-family: 'poppins', sans-serif;
+  width: 327px;
+  height: 56px;
+  line-height: 52%;
+}
+
+input[type="text"] {
+  width: 327px;
+  height: 48px;
+  border-radius: 8px;
+  background-color: #fff;
+  color: #172b4d;
+  text-align: left;
+}
+
+input[type="email"] {
+  width: 327px;
+  height: 48px;
+  border-radius: 8px;
+  background-color: #fff;
+  color: #172b4d;
+  text-align: left;
+}
+
+textarea {
+  width: 327px;
+  height: 170px;
+  border-radius: 8px;
+  color: #172b4d;
+  background-color: #fff;
+}
+
+input[type="submit"] {
+  width: 140px;
+  height: 48px;
+  border-radius: 8px;
+  padding: 12px 16px 12px 16px;
+  color: #6070ff;
+  font-family: 'poppins', sans-serif;
+  font-weight: 400;
+  font-size: 15px;
+}
+
+::placeholder[type="name"] {
+  color: #172b4d;
+  font-weight: 400;
+  font-size: 17px;
+}
+
+::placeholder[type="email"] {
+  color: #172b4d;
+  font-weight: 400;
+  font-size: 17px;
+}
+
+input[value] {
```

Comment:
- [ ] I kindly suggest that you add the different button interactions including hover, active (when it's pressed), disabled, and enable (default state) as shown in the Figma template below:

![Screenshot from 2022-04-27 19-12-50](https://user-images.githubusercontent.com/52934128/165592912-7d8bf224-8e6b-467d-bb19-cf63cd2b40fd.png)

the button on the pressed state has the `background: #2230D2;` and on hover has the `background: #6070FF;`

### EXAMPLE:
**for pressed state**
```
button:active {
   background-color: #2230D2;
}
```
**for hover state**
```
button:hover {
    background-color: #6070FF;
}
```